src/NHSD.BuyingCatalogue.Solutions.API/ViewModels/SolutionDashboardResult.cs

```diff
@@ -50,6 +50,8 @@ public class SolutionDashboardSections
         [JsonProperty("private-cloud")]
         public DashboardSection PrivateCloudSection { get; }
 
+        [JsonProperty("hybrid")]
```

Comment:
This has been changed in the JSON contract, could we update it please.

Reply:
updated